Framework/Muon/src/EstimateMuonAsymmetryFromCounts.cpp

```diff
@@ -207,11 +208,21 @@ void EstimateMuonAsymmetryFromCounts::exec() {
   }
   // update table
   Mantid::API::ITableWorkspace_sptr table = getProperty("NormalizationTable");
-  updateNormalizationTable(table, wsNames, norm, methods);
-  setProperty("NormalizationTable", table);
+  if (table) {
```

Comment:
if this is to be kept as optional then we will need to update the CalculateMuonAsymmetry to:

if no table is provided create a dict (or some other useful object) of the normalizations to use in the algorithm. If some information is missing it should be caught in the validation.

We can do this later as the calculate is not used until we do the data analysis tab. However, we will need an issue to ensure it is not forgotten.

Reply:
I've created an issue for this here #23803 let me know what other information needs to be added to it